### ISSUE TITLE
:analyze-path and other REPL options

### ISSUE THREAD

**Issue by pkobrien** [adzerk-oss/boot-cljs-repl#22]
Could we/ should we add support for :analyze-path and other REPL options described here:

https://github.com/clojure/clojurescript/wiki/REPL-options

**Comment by pkobrien** [comment]
Good info here: http://blog.fikesfarm.com/posts/2015-06-10-analyze-path-ftw.html

**Comment by Deraen** [comment]
The problem with analyze path is that cljs sources are not necessarily in one path. But we can create a tmpdir to which we copy all the cljs files (and cljc and clj?). I guess this needs to be done only once, at the start up.

**Comment by Deraen** [comment]
If I remember correctly, boot-cljs, copies all files in fileset to one directory before the build. So this would be easier to implement once boot-cljs-repl has been merged into boot-cljs.

**Comment by pkobrien** [comment]
Is boot-cljs-repl going to be merged into boot-cljs?
